src/main/kotlin/no/nav/syfo/infrastructure/kafka/senoppfolging/SenOppfolgingVarselConsumer.kt

```diff
@@ -30,8 +30,15 @@ class SenOppfolgingVarselConsumer(private val senOppfolgingService: SenOppfolgin
     }
 
     private fun processRecord(senOppfolgingVarselRecord: KSenOppfolgingVarselDTO) {
-        val existing = senOppfolgingService.findKandidatFromVarselId(senOppfolgingVarselRecord.uuid)
-        if (existing == null) {
+        val existingKandidatForVarsel = senOppfolgingService.findKandidatFromVarselId(varselId = senOppfolgingVarselRecord.uuid)
+        if (existingKandidatForVarsel == null) {
+            val recentKandidat = senOppfolgingService.findRecentKandidatFromPersonIdent(
+                personident = Personident(senOppfolgingVarselRecord.personident)
+            )
+            if (recentKandidat != null) {
+                log.warn("Found recent kandidat for person with uuid ${recentKandidat.uuid} - creating possible duplicate")
```

Comment:
Burde det være error? Vet ikke helt hvor vi har lagt listen i loggingen vår 😅 For dette er vel noe vi ønsker å agere på så snart som mulig etter det har skjedd 🤔

Reply:
Hmja bør kanskje det. Inntil vi eller esyfo har en håndtering av det på plass så må vi jo rydde opp. Endrer :+1: